### ISSUE TITLE
Electron wallet build failed for mac

### ISSUE THREAD

**Issue by iketheadore** [SkycoinProject/skycoin#225]
The travis failed in building electron wallet for mac:

Error message:
```
• electron-builder version=20.13.3
  • loaded configuration file=package.json ("build" field)
  • no native production dependencies
  • packaging       platform=darwin arch=x64 electron=7.1.7 appOutDir=release/mac
Error: ENOENT: no such file or directory, open '/Users/travis/gopath/src/github.com/SkycoinProject/skycoin/electron/release/mac/Electron.app/Contents/Frameworks/Electron Helper EH.app/Contents/Info.plist'
    at _bluebirdLst.default.map.it (/Users/travis/gopath/src/github.com/SkycoinProject/skycoin/electron/node_modules/electron-builder-lib/src/electron/electronMac.ts:38:215)
    at runCallback (timers.js:705:18)
    at tryOnImmediate (timers.js:676:5)
    at processImmediate (timers.js:658:5)
From previous event:
    at /Users/travis/gopath/src/github.com/SkycoinProject/skycoin/electron/node_modules/electron-builder-lib/src/electron/electronMac.ts:38:61
    at Generator.next (<anonymous>)
From previous event:
    at createMacApp (/Users/travis/gopath/src/github.com/SkycoinProject/skycoin/electron/node_modules/electron-builder-lib/out/electron/electronMac.js:218:17)
    at /Users/travis/gopath/src/github.com/SkycoinProject/skycoin/electron/node_modules/electron-builder-lib/src/electron/ElectronFramework.ts:56:11
    at Generator.next (<anonymous>)
From previous event:
    at beforeCopyExtraFiles (/Users/travis/gopath/src/github.com/SkycoinProject/skycoin/electron/node_modules/electron-builder-lib/out/electron/ElectronFramework.js:167:17)
    at beforeCopyExtraFiles (/Users/travis/gopath/src/github.com/SkycoinProject/skycoin/electron/node_modules/electron-builder-lib/src/electron/ElectronFramework.ts:127:14)
    at /Users/travis/gopath/src/github.com/SkycoinProject/skycoin/electron/node_modules/electron-builder-lib/src/platformPackager.ts:223:13
    at Generator.next (<anonymous>)
    at runCallback (timers.js:705:18)
    at tryOnImmediate (timers.js:676:5)
    at processImmediate (timers.js:658:5)
From previous event:
    at MacPackager.doPack (/Users/travis/gopath/src/github.com/SkycoinProject/skycoin/electron/node_modules/electron-builder-lib/out/platformPackager.js:410:7)
    at /Users/travis/gopath/src/github.com/SkycoinProject/skycoin/electron/node_modules/electron-builder-lib/src/macPackager.ts:88:63
    at Generator.next (<anonymous>)
From previous event:
    at MacPackager.pack (/Users/travis/gopath/src/github.com/SkycoinProject/skycoin/electron/node_modules/electron-builder-lib/out/macPackager.js:254:7)
    at /Users/travis/gopath/src/github.com/SkycoinProject/skycoin/electron/node_modules/electron-builder-lib/src/packager.ts:373:24
    at Generator.next (<anonymous>)
    at xfs.stat (/Users/travis/gopath/src/github.com/SkycoinProject/skycoin/electron/node_modules/fs-extra-p/node_modules/fs-extra/lib/mkdirs/mkdirs.js:56:16)
    at /Users/travis/gopath/src/github.com/SkycoinProject/skycoin/electron/node_modules/graceful-fs/polyfills.js:287:18
    at FSReqWrap.oncomplete (fs.js:154:5)
From previous event:
    at Packager.doBuild (/Users/travis/gopath/src/github.com/SkycoinProject/skycoin/electron/node_modules/electron-builder-lib/out/packager.js:615:7)
    at /Users/travis/gopath/src/github.com/SkycoinProject/skycoin/electron/node_modules/electron-builder-lib/src/packager.ts:317:52
From previous event:
    at Packager._build (/Users/travis/gopath/src/github.com/SkycoinProject/skycoin/electron/node_modules/electron-builder-lib/out/packager.js:542:7)
    at /Users/travis/gopath/src/github.com/SkycoinProject/skycoin/electron/node_modules/electron-builder-lib/src/packager.ts:290:23
    at Generator.next (<anonymous>)
    at runCallback (timers.js:705:18)
    at tryOnImmediate (timers.js:676:5)
    at processImmediate (timers.js:658:5)
From previous event:
    at Packager.build (/Users/travis/gopath/src/github.com/SkycoinProject/skycoin/electron/node_modules/electron-builder-lib/out/packager.js:501:7)
    at /Users/travis/gopath/src/github.com/SkycoinProject/skycoin/electron/node_modules/electron-builder-lib/src/index.ts:51:40
    at Generator.next (<anonymous>)
From previous event:
    at build (/Users/travis/gopath/src/github.com/SkycoinProject/skycoin/electron/node_modules/electron-builder-lib/out/index.js:220:17)
    at build (/Users/travis/gopath/src/github.com/SkycoinProject/skycoin/electron/node_modules/electron-builder/src/builder.ts:234:10)
    at then (/Users/travis/gopath/src/github.com/SkycoinProject/skycoin/electron/node_modules/electron-builder/src/cli/cli.ts:43:10)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! skycoin@0.27.0 dist-mac: `build -m --x64 -p=never`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the skycoin@0.27.0 dist-mac script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/travis/.npm/_logs/2020-02-22T01_36_14_135Z-debug.log
The command "if [[ "$TRAVIS_PULL_REQUEST" == false ]] && [[ "$TRAVIS_BRANCH" =~ ^(master|develop)$ || -n  "$TRAVIS_TAG" ]]; then ./ci-scripts/build-wallet.sh; fi" exited with 1.

```

**Comment by iketheadore** [comment]
PR #216 should be able to fix it.

**Comment by iketheadore** [comment]
Fixed by #216
